src/LibraryManager.Vsix/Json/Completion/FilesCompletionProvider.cs

```diff
@@ -29,6 +29,8 @@ public override JSONCompletionContextType ContextType
 
         protected override IEnumerable<JSONCompletionEntry> GetEntries(JSONCompletionContext context)
         {
+            Telemetry.TrackUserTask("CompletionForFiles");
```

Comment:
Is GetEntries() invoked by user ? If not, TrackOperation() is better.
Also modify the name to "verb-noun" format.